eslint-config-base/package.json

```diff
@@ -21,14 +21,14 @@
   },
   "homepage": "https://github.com/inrupt/javascript-style-configs#readme",
   "peerDependencies": {
-    "@typescript-eslint/eslint-plugin": "^3.0.1",
-    "@typescript-eslint/parser": "^3.0.1",
-    "eslint": "^7.1.0",
-    "eslint-config-airbnb-base": "^14.1.0",
+    "@typescript-eslint/eslint-plugin": "^5.6.0",
+    "@typescript-eslint/parser": "^5.6.0",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
     "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-header": "^3.1.1",
     "eslint-plugin-import": "^2.20.2",
     "eslint-plugin-jest": "^23.13.2",
-    "eslint-plugin-header": "^3.1.1",
     "eslint-plugin-prettier": "^3.1.3",
     "typescript": "*"
```

Comment:
Should we also pin typescript?

Reply:
We could pin it to versions supported by the linting rules, but those rules will already yell at you about a mismatched peer version of typescript. We'd just be updating our version to match those unless we're intentionally specifying a version of TS, and that's something I don't have a strong opinion on.